src/algebraic_geometry/open_immersion.lean

```diff
@@ -1498,6 +1509,51 @@ end
 instance forget_to_Top_preserves_of_right :
   preserves_limit (cospan g f) Scheme.forget_to_Top := preserves_pullback_symmetry _ _ _
 
+lemma range_pullback_snd_of_left :
+  set.range (pullback.snd : pullback f g ⟶ _).1.base =
```

Comment:
```suggestion
  set.range (pullback.snd : pullback f g ⟶ Y).1.base =
```
doesn't hurt to write it explicitly when it's just one letter; same below (3 occasions).